pkg/actors/placement/placement.go

```diff
@@ -157,6 +157,18 @@ func (p *actorPlacement) AddHostedActorType(actorType string, idleTimeout time.D
 	return nil
 }
 
+// Delete an actor type from the list of known actor types (if it's already registered)
+// The placement tables will get updated when the next heartbeat fires
+func (p *actorPlacement) DeleteHostedActorType(actorType string) error {
+	for i, t := range p.actorTypes {
```

Comment:
Shouldn't manipulating `actorTypes` be done under a lock, considering it's used for internal actors too, so it can be called concurrently?

Reply:
you are right, I'm adding a mutex protecting every use of `p.actorTypes`